presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2165,10 +2171,13 @@ public void createView(ConnectorSession session, ConnectorTableMetadata viewMeta
                 .build();
 
         List<Column> columns = new ArrayList<>();
+        MetastoreContext metastoreContext = new MetastoreContext(session.getIdentity(), session.getQueryId(), session.getClientInfo(), session.getSource(), getMetastoreHeaders(session), isUserDefinedTypeEncodingEnabled(session), columnConverterProvider);
+        ColumnConverter columnConverter = metastoreContext.getColumnConverter();
+
         for (ColumnMetadata column : viewMetadata.getColumns()) {
             try {
-                // issue #17081: propagate type metadata
```

Comment:
While we are removing this comment here, there is another one in `createTemporaryTable`.  Temp tables are not visible to users. So we can confidently having the primary types. But would be good to replace the comment there to emphasize temp tables do not need user defined types.